supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -192,16 +193,9 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     new JesExpressionFunctions(List(gcsFileSystem), callContext)
   }
 
-  private lazy val knownInputs = {
-    val unqualifiedWorkflowInputs = workflowDescriptor.inputs map {
-      case (fqn, v) => splitFqn(fqn)._2 -> v
-    }
-    unqualifiedWorkflowInputs ++ jobDescriptor.inputs
-  }
-
   private val lookup: ScopedLookupFunction = {
     val declarations = workflowDescriptor.workflowNamespace.workflow.declarations ++ call.task.declarations
-    WdlExpression.standardLookupFunction(knownInputs, declarations, callEngineFunctions)
+    WdlExpression.standardLookupFunction(jobDescriptor.inputs, declarations, callEngineFunctions)
```

Comment:
Instead of just `jobDescriptor.inputs`, this line previously used the `knownInputs`, a combo of `unqualifiedWorkflowInputs ++ jobDescriptor.inputs`.

I am _far_ from an expert on the lookup functions, so maybe I'm off target, but that's what was there before. The construction of the `val unqualifiedWorkflowInputs` could be added inside this block defining `val lookup`.